fbw-a32nx/src/wasm/radalt_a320/src/radaltControl.h

```diff
@@ -0,0 +1,617 @@
+	case SIMCONNECT_EXCEPTION_TOO_MANY_MAPS:			//10
+		return "EXCEPTION_TOO_MANY_MAPS (10)";
+	case SIMCONNECT_EXCEPTION_TOO_MANY_OBJECTS:			//11
+		return "EXCEPTION_TOO_MANY_OBJECTS (11)";
+	case SIMCONNECT_EXCEPTION_TOO_MANY_REQUESTS:				//12
+		return "EXCEPTION_TOO_MANY_REQUESTS (12)";
+	case SIMCONNECT_EXCEPTION_WEATHER_INVALID_PORT:				//13
+		return "EXCEPTION_WEATHER_INVALID_PORT (13)";
+	case SIMCONNECT_EXCEPTION_WEATHER_INVALID_METAR:			//14
+		return "EXCEPTION_WEATHER_INVALID_METAR (14)";
+	case SIMCONNECT_EXCEPTION_WEATHER_UNABLE_TO_GET_OBSERVATION://15
+		return "EXCEPTION_WEATHER_UNABLE_TO_GET_OBSERVATION (15)";
+	case SIMCONNECT_EXCEPTION_WEATHER_UNABLE_TO_CREATE_STATION:	//16
+		return "EXCEPTION_WEATHER_UNABLE_TO_CREATE_STATION (16)";
+	case SIMCONNECT_EXCEPTION_WEATHER_UNABLE_TO_REMOVE_STATION:	//17
+		return "EXCEPTION_WEATHER_UNABLE_TO_REMOVE_STATION (17)";
+	case SIMCONNECT_EXCEPTION_INVALID_DATA_TYPE:				//18
+		return "EXCEPTION_INVALID_DATA_TYPE (18)";
+	case SIMCONNECT_EXCEPTION_INVALID_DATA_SIZE:				//19
+		return "EXCEPTION_INVALID_DATA_SIZE (19)";
+	case SIMCONNECT_EXCEPTION_DATA_ERROR:						//20
+		return "EXCEPTION_DATA_ERROR (20)";
+	case SIMCONNECT_EXCEPTION_INVALID_ARRAY:			//21
+		return "EXCEPTION_INVALID_ARRAY (21)";
+	case SIMCONNECT_EXCEPTION_CREATE_OBJECT_FAILED:		//22
+		return "EXCEPTION_CREATE_OBJECT_FAILED (22)";
+	case SIMCONNECT_EXCEPTION_LOAD_FLIGHTPLAN_FAILED:	//23
+		return "EXCEPTION_LOAD_FLIGHTPLAN_FAILED (23)";
+	case SIMCONNECT_EXCEPTION_OPERATION_INVALID_FOR_OBJECT_TYPE://24
+		return "EXCEPTION_OPERATION_INVALID_FOR_OBJECT_TYPE (24)";
+	case SIMCONNECT_EXCEPTION_ILLEGAL_OPERATION:				//25
+		return "EXCEPTION_ILLEGAL_OPERATION (25)";
+	case SIMCONNECT_EXCEPTION_ALREADY_SUBSCRIBED:				//26
+		return "EXCEPTION_ALREADY_SUBSCRIBED (26)";
+	case SIMCONNECT_EXCEPTION_INVALID_ENUM:						//27
+		return "EXCEPTION_INVALID_ENUM (27)";
+	case SIMCONNECT_EXCEPTION_DEFINITION_ERROR:	//28
+		return "EXCEPTION_DEFINITION_ERROR (28)";
+	case SIMCONNECT_EXCEPTION_DUPLICATE_ID:		//29
+		return "EXCEPTION_DUPLICATE_ID (29)";
+	case SIMCONNECT_EXCEPTION_DATUM_ID:			//30
+		return "EXCEPTION_DATUM_ID (30)";
+	case SIMCONNECT_EXCEPTION_OUT_OF_BOUNDS:	//31
+		return "EXCEPTION_OUT_OF_BOUNDS (31)";
+	case SIMCONNECT_EXCEPTION_ALREADY_CREATED:	//32
+		return "EXCEPTION_ALREADY_CREATED (32)";
+	case SIMCONNECT_EXCEPTION_OBJECT_OUTSIDE_REALITY_BUBBLE://33
+		return "EXCEPTION_OBJECT_OUTSIDE_REALITY_BUBBLE (33)";
+	case SIMCONNECT_EXCEPTION_OBJECT_CONTAINER:				//34
+		return "EXCEPTION_OBJECT_CONTAINER (34)";
+	case SIMCONNECT_EXCEPTION_OBJECT_AI:					//35
+		return "EXCEPTION_OBJECT_AI (35)";
+	case SIMCONNECT_EXCEPTION_OBJECT_ATC:					//36
+		return "EXCEPTION_OBJECT_ATC (36)";
+	case SIMCONNECT_EXCEPTION_OBJECT_SCHEDULE:				//37
+		return "EXCEPTION_OBJECT_SCHEDULE (37)";
+	default:
+		return "UNKNOWN EXCEPTION";
+	}
+}
+void CALLBACK RadaltDispatchProc(SIMCONNECT_RECV* pData, DWORD cbData, void* pContext)
+{
```

Comment:
```suggestion
void CALLBACK RadaltDispatchProc(SIMCONNECT_RECV* pData, DWORD cbData, void* pContext) {
```
etc (more below but not tagged)